### PR TITLE
Extension now usable on Flipkart and snapdeal also.

### PR DESCRIPTION
**Pull request by dineshmakkar079**
Update the .crx file on your master branch also. I have not updated that.

### CODE REVIEW

File: js/myscript.js [hunk 1]
```diff
@@ -1,28 +1,83 @@
...
-    
-    i13 = $("li:contains('ISBN-13')").first().text().replace("ISBN-13:","").trim();
-    
-    name = $("#productTitle").first().text().trim();
-    // Alternates when viewing kindle books
-    if(name == "")
-    {
-        name = $("#ebooksProductTitle").first().text().trim();
+    var hostname = location.hostname.toLowerCase();
+    var hostcode = "N";
```
Comment (samj1912) [inline js/myscript.js]: Please dont keep hostname values as arbitrary constants. Define them somewhere and use enum or use verbose string names

**Comment by dineshmakkar079** [comment]
Made those changes. Did you request only these changes?



  <https://mailtrack.io/> Sent with Mailtrack
<https://mailtrack.io/install?source=signature&lang=en&referral=dineshmakkar079@gmail.com&idSignature=22>

File: js/myscript.js [hunk 2]
```diff
@@ -1,28 +1,92 @@
...
+                if(str.indexOf("ISBN") !== -1) 
+                {
+                    i13 = str.split(":")[1].split(",")[0].trim();
+                    break;
+                } 
+            }
+        }else if(hostcode === SNAPDEAL.hostcode){
+            let infoList = $(".h-content");
+            let len = infoList.length;
+            name = document.getElementsByClassName("pdp-e-i-head")[0].innerText;
```
Comment (samj1912) [inline js/myscript.js]: Use jquery for all of dom selections

File: js/myscript.js [hunk 3]
```diff
@@ -1,28 +1,92 @@
...
+    const INVALID = "INVALID"; 
     
-    i13 = $("li:contains('ISBN-13')").first().text().replace("ISBN-13:","").trim();
-    
-    name = $("#productTitle").first().text().trim();
-    // Alternates when viewing kindle books
-    if(name == "")
-    {
-        name = $("#ebooksProductTitle").first().text().trim();
+    var hostcode = NOT_VALID.hostcode;
```
Comment (samj1912) [inline js/myscript.js]: Not able to understand the logic of doing a case switch here and then again going through an if else later based on host code. Can't you just do the indexOf comparisons in L21(if/else block) and so on?

**Comment by samj1912** [comment]
Cool, this LGTM. If you are further interested in contributing, you might write some tests and I can help you with setting up a build system for this repo to automatically run the tests and package the code into an extension.

**Comment by dineshmakkar079** [comment]
I didn't get you and LGTM also
Can you elaborate more on what you are trying to say.
And thanks for investing your time in reviewing code multiple times.
Are you satisfied with this now?



  <https://mailtrack.io/> Sent with Mailtrack
<https://mailtrack.io/install?source=signature&lang=en&referral=dineshmakkar079@gmail.com&idSignature=22>

**Pull request merged**